src/content/api/module-methods.md

```diff
@@ -109,6 +114,10 @@ W> Note that setting `webpackIgnore` to `true` opts out of code splitting.
 - `"eager"`: Generates no extra chunk. All modules are included in the current chunk and no additional network requests are made. A `Promise` is still returned but is already resolved. In contrast to a static import, the module isn't executed until the call to `import()` is made.
 - `"weak"`: Tries to load the module if the module function has already been loaded in some other way (i. e. another chunk imported it or a script containing the module was loaded). A `Promise` is still returned but, only successfully resolves if the chunks are already on the client. If the module is not available, the `Promise` is rejected. A network request will never be performed. This is useful for universal rendering when required chunks are always manually served in initial requests (embedded within the page), but not in cases where app navigation will trigger an import not initially served.
 
+`webpackPrefetch`:  This tells the browser that the resource is probably needed for some navigation in the future. Check out the guide for more information on [how webpackPrefetch works](https://webpack.js.org/guides/code-splitting/#prefetching-preloading-modules).
```

Comment:
Please remove double space after colon

Reply:
Please use relative URLs: `/guides/code-splitting/#prefetching-preloading-modules`

Reply:
Same comments apply to `webpackPreload`

Reply:
`This tells the...` "this" is redundant here imo, you are describing `webpackPrefetch` after a colon

Reply:
done